src/components/ReceiveMyAddress.js

```diff
@@ -43,8 +47,9 @@ class ReceiveMyAddress extends React.Component {
   }
 
   getNextAddress = () => {
-    const nextAddress = hathorLib.wallet.nextAddress();
-    this.setState({ address: nextAddress });
+    // Mark current address as used, then get the next
+    this.props.wallet.getCurrentAddress({ markAsUsed: true });
+    this.setState({ address: this.props.wallet.getCurrentAddress() });
```

Comment:
Should we create a `getNextAddress()` method?

Reply:
I agree with you. Even though I am not 100% sure about this, if we keep this feature in the wallet we should have this method

Reply:
Done

Reply:
Great! Thanks!